crates/autopilot/src/run.rs

```diff
@@ -400,8 +401,9 @@ pub async fn run(args: Arguments) {
         .unwrap();
     let native_price_estimator = price_estimator_factory
         .native_price_estimator(
+            args.native_price_estimators.as_slice(),
             &PriceEstimatorSource::for_args(
-                args.native_price_estimators.as_slice(),
+                args.order_quoting.price_estimators.as_slice(),
```

Comment:
This change effectively means that all price estimators are automatically native price estimators. This wasn’t exactly what I had in mind (it was more that I didn’t think that `GenericPriceEstimator` variant made sense).

Reply:
They should only be in the reportoire of possible sources. The source of truth of which estimators get chosen via the `--native-price-estimators` list (ie we no longer automatically add all external price estimators as native estimators)